.travis.yml

```diff
@@ -3,14 +3,27 @@ language: python
 matrix:
   fast_finish: true
   include:
+  - python: 3.7  # Separate build to run flake8 check
+    env:
+      - INTAKE_MASTER=false
+      - FLAKE_8=true
   - python: 3.6
-    env: INTAKE_MASTER=false
+    env:
+      - INTAKE_MASTER=false
+      - FLAKE8=
```

Comment:
This variable name is different from the one which has `true`. I suggest removing the ones which have empty values, as they will default to `false`. I suggested to do the same in https://github.com/bluesky/ophyd/pull/823/files#r394048095. I personally don't like the `_`-separation between `FLAKE` and `8`. `FLAKE8` would match the corresponding program name. But it should be coordinated with @dmgav in his flake8 PR in ophyd.

Reply:
Or it can be called something like `CHECK_STYLE` to open a potential for another tool in the future.

Reply:
I replaced `FLAKE_8` by `FLAKE8` in the `ophyd` PR.